.github/workflows/package-Instrumentation.AspNet.yml

```diff
@@ -55,13 +55,13 @@ jobs:
         nuget push **/${{env.PROJECT}}/bin/**/*.nupkg -Source https://api.nuget.org/v3/index.json -ApiKey ${{ secrets.NUGET_TOKEN }} -SymbolApiKey ${{ secrets.NUGET_TOKEN }}
 
     - name: Create GitHub Prerelease
-      if: ${{ (contains(github.ref_name, '-alpha.') || contains(github.ref_name, '-beta.') || contains(github.ref_name, '-rc.')) }}
+      if: ${{ (contains(github.ref_name, '-alpha.') || contains(github.ref_name, '-beta.') || contains(github.ref_name, '-rc.') || contains(github.ref_name, '-rc9.')) }}
```

Comment:
Why not update the versions of these packages to `rc.` to make it consistent?

Reply:
Issue occurred before migration from main repository https://github.com/open-telemetry/opentelemetry-dotnet/pull/3108

To fix it we should release these 3 packages as at least 1.0.1-beta.1/rc.1 or any higher version. Otherwise it will be treated as older version.

Reply:
Oh I just realized that we had also changed the package names when did the version name change for the other packages.